src/asciidoctor-core-api.js

```diff
@@ -292,14 +326,22 @@ Opal.Asciidoctor.Document.$$proto.setTitle = function (title) {
   return this['$title='](title);
 };
 
-Opal.Asciidoctor.Document.$$proto.getDoctitle = function (options) {
+Opal.Asciidoctor.Document.$$proto.getDoctitle = Opal.Asciidoctor.Document.$$proto.getDocumentTitle = function (options) {
   return this.$doctitle(toHash(options));
 };
 
-Opal.Asciidoctor.Document.$$proto.getRevdate = function () {
+Opal.Asciidoctor.Document.$$proto.getRevdate = Opal.Asciidoctor.Document.$$proto.getRevisionDate = function () {
   return this.$revdate();
 };
 
+Opal.Asciidoctor.Document.$$proto.getRevnumber = Opal.Asciidoctor.Document.$$proto.getRevisionNumber = function () {
+  return this.getAttributes().revnumber;
```

Comment:
@mojavelinux Do you think Revnumber and Revremark should be in the Asciidoctor core API ?